msteams-platform/overview.md

```diff
@@ -1,100 +1,55 @@
-
-Personal tabs allow individual users to work privately with your tools and data, in their own dedicated app space.  Use these tabs to surface information catering to the individual such as personal task lists or dashboards, or provide help and support information.
-
----
-
-[Get started with tabs.](~/concepts/tabs/tabs-overview)
-
----
-
-![Example of a tab showing data, alongside a conversation about the tab data](~/assets/images/tab_example.png)
-
-### Bots
-
-Build and connect intelligent bots to interact with Microsoft Teams users naturally through chat. Bots can answer natural-language questions and perform lightweight tasks such as querying bug information or kicking off a build.​ Any bot you've created using the [Microsoft Bot Framework](https://dev.botframework.com/) can easily be made to work in Microsoft Teams.
-
-While bots can take advantage of the rich natural-language processing provided by the Bot Framework, your bot can also act as the command-line interface of your experience.  Or you can choose to make a notification-only bot, which can push information relevant to your users directly to them in a channel or direct message.
+> [!TIP]
+> Don't have Microsoft Teams? Get a free Office 365 developer subscription or activate it for your existing Office 365 account. See [Set up your Office 365 tenant](~/get-started/get-started#1-set-up-your-office-365-tenant).
 
----
-
-[Get started with bots.](~/concepts/bots/bots-overview)
-
----
-
-![Example of a bot assisting a user](~/assets/images/bot_example.png)
+## What you'll find in these topics
 
-### Compose extensions
+Are you new to the Microsoft Teams developer platform? Choose one of the following walkthroughs to quickly learn the basics of developing an app for Microsoft Teams.
 
->[Public Developer Preview only](~/reference/general/developer-preview)
+*	[Get started with .NET and C#](~/get-started/get-started-dotnet)
+*	[Get started with Node.js](~/get-started/get-started-nodejs)
 
-Leverage the power of your web service to provide a quick and easy way for your users to insert your content into the chat stream.
-
----
-
-[Get started with compose extensions.](~/concepts/compose-extensions)
-
----
+The rest of this documentation provides the information you need to extend Microsoft Teams by adding your content and services.
 
-![Example of a compose extension](~/assets/images/composeextension/ceoverviewexample.png)
+* **Concepts**&emsp;Learn more about creating Teams apps, and find everything you need to know about the entire range of capabilities in Teams: tabs, bots, Connectors, compose extensions, and more.
+* **Publishing**&emsp;After you develop your app, you need to test and publish it. These topics explain how.
+* **Scenarios**&emsp;Go deep into the specifics of conversations with bots.
+* **Reference**&emsp;Learn how to design a great Teams app, get the details about our SDKs, and more.
```

Comment:
I would say 'Come back and check a specific topic, API or a message structure. You will find all the gory details you need for building your apps'

Reply:
I'll leave the gore here and go for the nitty-gritty. :-D